File: LLM-Translations/GPT-4/codenet/C++/Go/s984540390.go

package main

import (
	"fmt"
	"math"
)

type ll int64

func main() {
	var h, ans, mx, sum, cnt ll
	mn := ll(math.Pow10(8))
	cnt = 1

	fmt.Scan(&h)
	for h != 1 {
		if h%2 != 0 {
			h--
		}
		h /= 2
		cnt *= 2
		ans += cnt
	}
	fmt.Println(ans + 1)
}

// 

File: LLM-Translations/GPT-4/codenet/C++/Python/s984540390.py

import sys
from math import log2, ceil

def main():
    h = int(input().strip())
    ans, cnt = 0, 1

    while h != 1:
        if h % 2 != 0:
            h -= 1
        h //= 2
        cnt *= 2
        ans += cnt

    print(ans + 1)

if __name__ == "__main__":
    main()

#
